### PR TITLE
Copying does not contain the names or "Image - " for single messages

### PR DESCRIPTION
**Pull request by Hocuri**
We have replies for this now

### CODE REVIEW

**Review by adbenitez — APPROVED** [review]
tested, great!

File: src/org/thoughtcrime/securesms/ConversationFragment.java [hunk 1]
```diff
@@ -415,11 +416,11 @@ private void handleCopyMessage(final Set<DcMsg> dcMsgsSet) {
...
-            if (msg.getFromId() != prevMsg.getFromId()) {
+            if (msg.getFromId() != prevMsg.getFromId() && !singleMsg) {
                 DcContact contact = dcContext.getContact(msg.getFromId());
                 result.append(contact.getDisplayName()).append(":\n");
             }
-            if(msg.getType() == DcMsg.DC_MSG_TEXT) {
+            if(msg.getType() == DcMsg.DC_MSG_TEXT || singleMsg) {
                 result.append(msg.getText());
             } else {
                 result.append(msg.getSummarytext(10000000));
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/ConversationFragment.java:426]: that would result in copying an empty text if the user tries to copy eg. an image or a voice-message.

before, the image type was copied to the clipboard in the these cases (from summary then).

therefore, we should append the result from msg.getText() only if not empty - else use the summary.

**Review by r10s — CHANGES_REQUESTED** [review]

File: src/org/thoughtcrime/securesms/ConversationFragment.java (hunk 1 above)
Comment (Hocuri) [reply src/org/thoughtcrime/securesms/ConversationFragment.java]: Don't know what the usecase of copying the word "Image" would be but anyway, done (I don't think it's an improvement but it doesn't make anything worse, either)

**Review by r10s — APPROVED** [review]
lgtm now :)

**Pull request merged**